polaris-react/src/components/AlphaStack/AlphaStack.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import type {spacing} from '@shopify/polaris-tokens';
+
+import {classNames} from '../../utilities/css';
+
+import styles from './Stack.scss';
+
+type SpacingTokenGroup = typeof spacing;
+type SpacingTokenName = keyof SpacingTokenGroup;
+
+// TODO: Bring this logic into tokens
+type Spacing = SpacingTokenName extends `space-${infer Scale}` ? Scale : never;
+
+type Align = 'start' | 'end' | 'center';
+
+export interface AlphaStackProps {
+  /** Elements to display inside stack */
+  children?: React.ReactNode;
+  /** Adjust spacing between elements */
+  spacing?: Spacing;
+  /** Adjust vertical alignment of elements */
+  align?: Align;
+}
+
+export const AlphaStack = ({
+  children,
+  spacing,
```

Comment:
```suggestion
  spacing = 'var(--p-space-4)',
```

If we do this we don't need to redeclare the property in css. This is personal preference and not a blocker. Ship whatever you feel is best